MangaManager/src/Common/loadedcomicinfo.py

```diff
@@ -82,6 +82,18 @@ class LoadedComicInfo:
     _new_backcover_path: str | None = None
     new_backcover_cache: ImageTk.PhotoImage | None = None
 
+    def get_cover_cache(self) -> ImageTk.PhotoImage | None:
+        if self._cover_action is None:
```

Comment:
What is the difference between new_ and non-new_? If you can add a comment on the variable, it would be helpful.